src/CodeStyle/Core/CodeFixes/Host/Mef/CodeStyleHostLanguageServices.cs.cs

```diff
@@ -2,16 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable disable
```

Comment:
I get this is a revert, but leave the #nullable enable in place, since clearly it was OK before?

Reply:
➡️ Intentionally left as a follow-up. Other corrections were made during the commit that was reverted.